spec/controllers/cases_controller/show_spec.rb

```diff
@@ -87,7 +89,7 @@
 
 
         it 'permitted_events to be add_message_to_case' do
-          expect(assigns(:permitted_events)).to eq [:add_message_to_case]
+          expect(assigns(:permitted_events)).to be_empty
```

Comment:
These spec descriptions are not matching the tests. We should just find a way to do this using `subject` and the `should-matchers` like we do with the policy specs.